### PR TITLE
Fix Form Issues

### PR DESCRIPTION
**Pull request by ofarrero**
Add redirect to create question page to avoid user error
Add function to set default preference form values to the current preference

### CODE REVIEW

File: src/main/webapp/createExam.html [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  <body>
+    <header>
+      <div class="navtop">
+        <a  href="dashboard.html">Dashboard</a>
+        <p id=logInOut></p>
+      </div>
+    </header>
+    <main>
+      <h2>Create Exam</h2>
+      <form id="makeExam" action="/test" method="POST" onsubmit="window.location.href = 'createQuestion.html'; return false;">
```
Comment (AidanMolloy) [inline src/main/webapp/createExam.html:19]: Where does /test go to?

File: src/main/webapp/createExam.html [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+        <label for="examName">Enter Exam Name:</label><br>
+        <input type="text" id="examName" name="examName" required><br>
```
Comment (AidanMolloy) [inline src/main/webapp/createExam.html:21]: should examName be replaced with name?

Comment (ofarrero) [reply src/main/webapp/createExam.html]: Updated to work with your changes

**Comment by AidanMolloy** [comment]
lgtm

**Pull request merged**